lib/Doctrine/ORM/Mapping/FieldMetadata.php

```diff
@@ -18,21 +17,10 @@ class FieldMetadata extends LocalColumnMetadata implements Property
     /** @var string */
     protected $name;
 
-    /**
-     * @param string $columnName
-     * @param Type   $type
-     *
-     * @todo Leverage this implementation instead of default, simple constructor
-     */
-    /*public function __construct(string $name, string $columnName, Type $type)
-    {
-        parent::__construct($columnName, $type);
-
-        $this->name = $name;
-    }*/
-
-    public function __construct(string $name)
+    public function __construct(string $name/*, string $columnName, Type $type*/)
```

Comment:
Weird one - is there an issue or WIP for this, @guilhermeblanco?

Reply:
WIP